### PR TITLE
Touch all the todos so we can review them

### PR DESCRIPTION
**Pull request by tristanmenzel**

### CODE REVIEW

File: src/puya/awst_build/validation/scratch_slots.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def invalid_slot_usages(self) -> Iterator[tuple[int, SourceLocation]]:
 
     def visit_contract_fragment(self, statement: awst_nodes.ContractFragment) -> None:
         super().visit_contract_fragment(statement)
-        # TODO: gather reserved from bases
+        # TODO : gather reserved from bases
```
Comment (tristanmenzel) [inline src/puya/awst_build/validation/scratch_slots.py:35]:
```suggestion
```

We are now doing this [here](https://github.com/algorandfoundation/puya/blob/a5466a0d9644b15049a567581f1081aa47868644/src/puya/awst_build/module.py#L195)

File: src/puya/codegen/teal.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def __str__(self) -> str:
 
 @attrs.frozen
 class Dig(TealOpN):
-    op_code: str = "dig"  # TODO: these should be field with default=... and init=False
+    op_code: str = "dig"  # TODO : these should be field with default=... and init=False
```
Comment (tristanmenzel) [inline src/puya/codegen/teal.py:78]: Resolved [here](https://github.com/algorandfoundation/puya/pull/73)

File: src/puya/arc32.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def _encode_state_schema(state: Sequence[ContractState]) -> JSONDict:
 def _encode_schema_declaration(state: ContractState) -> JSONDict:
     return {
         "type": state.storage_type.name,
-        "key": state.key.decode("utf-8"),  # TODO: support not utf8 keys?
+        "key": state.key.decode("utf-8"),  # TODO : support not utf8 keys?
```
Comment (daniel-makerx) [inline src/puya/arc32.py:40]: Can remove, Have a ticket for this

File: src/puya/arc32.py [hunk 4]
```diff
@@ -77,7 +77,7 @@ def _encode_default_arg(contract: CompiledContract, source: str, loc: SourceLoca
         if state.name == source:
             return {
                 "source": "global-state",
-                # TODO: handle non utf-8 bytes
+                # TODO : handle non utf-8 bytes
```
Comment (daniel-makerx) [inline src/puya/arc32.py:80]: Have ticket

File: src/puya/awst_build/contract.py [hunk 5]
```diff
@@ -426,7 +426,7 @@ def _gather_app_state(
                     source_location=var_loc,
                     member_name=name,
                     storage_wtype=storage_wtype,
-                    key=name.encode(),  # TODO: encode name -> key with source file encoding?
+                    key=name.encode(),  # TODO : encode name -> key with source file encoding?
```
Comment (daniel-makerx) [inline src/puya/awst_build/contract.py:429]: Have ticket

File: src/puya/errors.py [hunk 6]
```diff
@@ -38,7 +38,7 @@ class TodoError(CodeError):
     """Not a code error but it's not a crash either"""
 
     def __init__(self, location: SourceLocation | None, msg: str | None = None):
-        super().__init__(msg or "TODO: support this thing", location=location)
+        super().__init__(msg or "TODO : support this thing", location=location)
```
Comment (daniel-makerx) [inline src/puya/errors.py:41]: Should remove this error now

File: src/puya/ir/destructure/coalesce_locals.py [hunk 7]
```diff
@@ -47,7 +47,7 @@ def coalesce_registers(group_strategy: CoalesceGroupStrategy, sub: models.Subrou
     """
     vla = VariableLifetimeAnalysis.analyze(sub)
 
-    # TODO: this uses a basic definition of interference by looking at live-ranges,
+    # TODO : this uses a basic definition of interference by looking at live-ranges,
```
Comment (daniel-makerx) [inline src/puya/ir/destructure/coalesce_locals.py:50]: Have a ticket for this, but some of the context might be useful still as a comment for now

**Pull request closed without merging**